data/json/recipes/recipe_food.json

```
@@ -4687,7 +4687,7 @@
     "subcategory": "CSC_FOOD_OTHER",
     "skill_used": "cooking",
     "time": "1 m",
-    "autolearn": true,
+    "book_learn": [ [ "vinegar_maker", 1 ], [ "winemaking_beginner", 1 ] ],
```

```suggestion
    "autolearn": true,
```
This is just a dilution recipe and can be autolearned